src/components/DetailSidebar/DetailSidebarFundButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { MouseEvent } from 'react';
+import styled from '@emotion/styled';
+import Favorite from '@material-ui/icons/Favorite';
+
+import Button from '../../muiComponents/Button';
+import Link from '../Link';
+import { Theme } from '../../design-tokens/theme';
+
+const StyledLink = styled(Link)<{ theme?: Theme }>(({ theme }) => ({
+  marginTop: theme && theme.spacing(1),
+  marginBottom: theme && theme.spacing(1),
+  textDecoration: 'none',
+  display: 'block',
+}));
+
+const StyledFavoriteIcon = styled(Favorite)<{ theme?: Theme }>(({ theme }) => ({
+  color: theme && theme.palette.orange,
+}));
+
+const StyledFundStrong = styled('strong')({
+  marginRight: 3,
+});
+
+interface Props {
+  to: string;
+}
+
+/* eslint-disable react/jsx-no-bind */
+const DetailSidebarFundButton: React.FC<Props> = ({ to }) => {
+  const preventDefault = (event: MouseEvent<HTMLButtonElement>) => event.preventDefault();
```

Comment:
The user clicks the button and then what happens? .... We should open the target in a new tab.

Reply:
We must also update `@verdaccio/types` to be able to have

```
  "funding": {
    "type": "opencollective",
    "url": "https://opencollective.com/verdaccio"
  },
```
in https://github.com/verdaccio/monorepo/blob/master/core/types/index.d.ts#L132 as optional type. Then can be used here.

Reply:
You can find all the rules about that field here https://github.com/npm/rfcs/pull/54/files#diff-1bc199539d172d3211c84688a617e18cR30

Reply:
@juanpicado thank you for your feedback :) 

The component StyledLink has the prop "external". When this prop is set to true, the anchor element will have target="_blank" .😉

Reply:
so is it configurable from the verdaccio config file ?

I would make it default to new tab (\_blank)